TABScrollingContentViewTests/TABScrollingContentViewTests.swift

```diff
@@ -0,0 +1,49 @@
+//
```

Comment:
we should probably put the test files in a `Tests` folder to follow the Swift Package Manager convention as it looks like this project already follows it as I'm seeing the `Sources` folder